contributing_to_docs/doc_guidelines.adoc

```diff
@@ -42,29 +33,59 @@ Please set your editor to strip trailing whitespace.
 
 [NOTE]
 ====
-The topic title, which is the first line of the document, is the only level 1 ( = ) title. Section headers within the topic must be level 2 ( == ) or lower.
+The assembly title, which is the first line of the document, is the only level 1 ( = ) title.
+Section headers within the assembly must be level 2 ( == ) or lower. When you include modules, you must add
+leveloffsets in the include statements. You can manually add more level 2 or lower section headers in the assembly.
 ====
 
-*All titles must have a unique (within the OpenShift docs suite) section anchor*, and this anchor must be all lowercase letters, with no line spaces between the anchor and the section title:
+== Module file metadata
+Every module should be placed in the modules folder and should contain the following metadata at the top:
 
 ----
-[[section-anchor-name]]
-=== Section Title
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included              <1>
+
+[id='<module-anchor>-{context}']                                   <2>
+= Module title                                                     <3>
 ----
 
-== Topic File Names
+<1> List of assemblies in which this module is included
+<2> A module anchor with {context} that must match the module's file name.
+<3> Human readable title
```

Comment:
I'd probably add something like, "To ensure consistency in the results of the `leveloffset` values in include statements, you must use a level one heading ( = ) for the module title."